model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py

```diff
@@ -253,7 +246,7 @@ async def train(self, sources: Sources):
             )
             data.append(feature_data)
         vw_data = pd.DataFrame(data)
-        if self.parent.config.convert_to_vw:
+        if self.parent.config.requires_formatting:
```

Comment:
Following the above example

```suggestion
        if not self.parent.config.noconvert:
```